File: Array/21_30/26Solution.py
def sumZero(n):
        out = []            
        for x in range(1, (n//2)+1):
            out.append(x)
            out.append(-x)
        if n % 2 != 0: out.append(0)
        return out
print(sumZero(int(input())))

File: Array/21_30/26Solution.java
class Solution {
    public int[] sumZero(int n) {
        int[] arr=new int[n];
        int index=0;
        int start=n/2;
        while(start>0){
            arr[index++]=start;
            arr[index++]=start*-1;
            start--;
        }
        
        return arr;
    }
}
